### PR TITLE
mb/ar/documentation

### PR DESCRIPTION
**Pull request by mbeyeler**
* fixed placement of movies on intro page
* updated intro page description
* updated README.md
* excluded all relative imports from sphinx doc
* fixed all mis-renderings (I believe)

### CODE REVIEW

**Comment by arokem** [comment]
Sorry - I rebased my `documentation` branch on top of uwescience/master and pushed it up (to simplify the diff only to the things that are really different from `master`), and now it seems that we have conflicts (that I can't resolve). Why the PDF file would be conflicted, I don't quite understand, but hopefully the other conflicts are simple to fix.

**Comment by mbeyeler** [comment]
OK, this should do the trick!

File: pulse2percept/electrode2currentmap.py [hunk 1]
```diff
@@ -53,7 +52,7 @@ def gamma(n, tau, t):
 
     y = ((t / tau) ** (n - 1) *
          np.exp(-t / tau) /
-         (tau * factorial(n - 1)))
+         (tau * sp.misc.factorial(n - 1)))
```
Comment (arokem) [inline pulse2percept/electrode2currentmap.py:55]: I'm pretty sure this wouldn't work. Scipy is weird that way, in that it doesn't import it's sub-modules unless they are explicitly imported: 

    from scipy import misc as spm 
    spm.factorial

File: pulse2percept/electrode2currentmap.py [hunk 2]
```diff
@@ -843,15 +869,15 @@ def __init__(self, rflum, tsample, fps=30.0, amplitude_transform='linear',
                 ppt = np.concatenate((ppt, pulse), axis=0)
 
         ppt = ppt[0:round(dur / tsample)]
-        intfunc = interpolate.interp1d(np.linspace(0, len(rflum), len(rflum)),
-                                       rflum)
+        intfunc = sp.interpolate.interp1d(np.linspace(0, len(rflum),
```
Comment (arokem) [inline pulse2percept/electrode2currentmap.py:872]: Same comment here. I believe that you need to explicitly import the `interpolate` sub-module for this to work.

**Review by arokem — APPROVED** [review]
Looks great! I had a couple of small comments about scipy imports. I'll go ahead and merge to mine and make the changes on my branch, and then we can see whether everything is resolved on my branch.

**Pull request merged**